pkg/ddc/jindocache/sync_runtime.go

```diff
@@ -213,17 +215,34 @@ func (e *JindoCacheEngine) syncFuseSpec(ctx cruntime.ReconcileRequestContext, ru
 				e.Log.V(1).Info("The resource requirement of fuse is the same, skip")
 			}
 
+			metricsEnabled, exists := fusesToUpdate.Spec.Template.ObjectMeta.Annotations[common.AnnotationPrometheusFuseMetricsScrapeKey]
+			if exists && metricsEnabled != common.True {
```

Comment:
How about letting the code exit early if the annotation exists but is not equal to common.True, reducing nesting?

```golang
if exists && metricsEnabled != common.True {
    e.Log.V(1).Info(fmt.Sprintf("Found user-defined annotation %s != %s, skip syncing.", common.AnnotationPrometheusFuseMetricsScrapeKey, common.True))
    return
}
```

Reply:
@cheyang The nesting is necessary. L235-L248 updates FUSE's daemonset if `changed==true`. If the code exits early, the update will be skipped when `runtime.Spec.Fuse.Resources` is modified(at L213).